extensions/shared/external-media/sources/google-photos/google-photos-auth.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import requestExternalAccess from '@automattic/request-external-access';
```

Comment:
In other places it's under `Internal dependencies`. I think it makes sense since it's actually a local dependency?

Reply:
I checked for other instances of `@automattic/` imports and they're all external. It's an npm dependency after all